Kitodo/src/main/java/org/kitodo/production/services/command/KitodoScriptService.java

```diff
@@ -481,14 +476,11 @@ public void deleteTiffHeaderFile(List<Process> processes) {
         Helper.setMessage(KITODO_SCRIPT_FIELD, "", "deleteTiffHeaderFile finished");
     }
 
-    private void exportDms(List<Process> processes, String exportImages, boolean exportFulltext) {
```

Comment:
why is the fullText not exported anymore?

Reply:
The full text will be exported exactly if a folder is configured in the folder settings and if the switch for export is set to "on". This will delete the export of a hard-coded folder. If you want to continue exporting the folder, you must configure it.